GUACAMOLE-1289: Update Duo documentation for "Universal Prompt".

mike-jumper opened the pull request.

necouchman commented on src/duo-auth.md:
```diff
@@ -110,57 +110,46 @@ The application-specific configuration information retrieved from Duo
-duo-integration-key
+`duo-client-id`
 : The integration key provided for Guacamole by Duo. This value will
   have been generated by Duo when you added Guacamole as an "Web SDK"
   application, and can be found within the application details in the
-  "Integration key" field. *This value is required and must be EXACTLY
-  20 characters.*
+  "Client ID" field. *This value is required.*
```
* Is this still an "integration key", or should this first line be changed to something like `The client identifier provided for Guacamole...`?
* While we're here modifying this block, perhaps change "an" to "a": `a "Web SDK"`

mike-jumper replied:
Oops - will fix both of these.

necouchman commented on src/duo-auth.md:
```diff
@@ -110,57 +110,46 @@ The application-specific configuration information retrieved from Duo
+  in older versions of Guacamole.
+
+`duo-client-secret`
 : The secret key provided for Guacamole by Duo. This value will have
```
As with above - does it make sense to change this to `The client secret`?

necouchman reviewed (changes requested):
Just a couple of potential changes...

mike-jumper commented:
OK - I've looked through things and:

* Corrected all the wonky "an Web SDK" occurrences.
* Replaced old mentions of the integration and secret keys with their new names.

necouchman commented on src/duo-auth.md:
```diff
@@ -110,57 +109,46 @@ The application-specific configuration information retrieved from Duo
+  in older versions of Guacamole.
+
+`duo-client-secret`
+: The shared secret provided for Guacamole by Duo. This value is will have
```
Extra `is` in this line: `This value is will...`

mike-jumper replied:
Fixed via rebase.

necouchman commented on src/duo-auth.md:
```diff
@@ -94,11 +94,10 @@ prompted by Duo for additional authentication:
-integration key, secret key, and API hostname - they will later be
-specified within `guacamole.properties`:
+Once you've finished adding Guacamole as a "Web SDK" application, the
+configuration information required to configure Guacamole is listed within the
```
I wouldn't have worried about this if there weren't something else to change, but the `configuration information required to configure Guacamole` feels a bit redundant. Maybe just `the information required to configure Guacamole`?

mike-jumper replied:
Fixed via rebase.

necouchman reviewed (changes requested):
A couple of more tweaks...
